vipps-qr-api-faq.md

```diff
@@ -18,19 +16,15 @@ Document version 0.0.1.
 It contains answers to all(?) common questions about Vipps payments:
 [Vipps eCom API FAQ](https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api-faq.md).
 
-## Can we make our own QR code based on the Vipps deeplink URL?
-
-Technically: Yes.
+## Cant we make our own QRs for payment and redirects?
```

Comment:
Use `can't`, 'cant' means something different.
Or just use the original phrasing.

Reply:
Or even better - just "can" in this case 😅